<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setarePIN</class>
 <widget class="QDialog" name="setarePIN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Changer le code PIN</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background: rgb(168, 0, 0);
}</string>
  </property>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 28px;
}</string>
   </property>
   <property name="text">
    <string>Confirmer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="oldPINLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 16px;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPINLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 16px;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmPINLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 16px;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="oldPINLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 18px;
}</string>
   </property>
   <property name="text">
    <string> Ancien code PIN:</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirmPINLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 18px;
}</string>
   </property>
   <property name="text">
    <string> Confirme le code PIN:</string>
   </property>
  </widget>
  <widget class="QLabel" name="newPINLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 18px;
}</string>
   </property>
   <property name="text">
    <string> Nouveau code PIN:</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>332</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>oldPINLineEdit</tabstop>
  <tabstop>newPINLineEdit</tabstop>
  <tabstop>confirmPINLineEdit</tabstop>
  <tabstop>confirmButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
